pkg/polaris/polaris_test.go

```diff
@@ -105,7 +105,7 @@ func TestAwsAccountAddAndRemove(t *testing.T) {
 	// Add the default AWS account to Polaris. Usually resolved using the
 	// environment variables AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and
 	// AWS_DEFAULT_REGION.
```

Comment:
We should update the comment as well to reflect the change

Reply:
Correct. I'll update!